CONTRIBUTING.md

```diff
@@ -134,6 +135,40 @@ cd test/manual/k8s_crds
 ```
 This test currently does not run as part of the test suite.
 
+## Documentation
+Secretless has a few sources for documentation: a website, a documentation subdomain, and godocs.
+
+### Website
+The [website](https://secretless.io) source is in the [docs](docs/) folder in this repository. It is generated using Jekyll.
+
+#### Prerequisites
+
+To get the site up and running locally on your computer, ensure you have:
+1. Ruby version 2.1.0 or higher (check by running `ruby -v`)
+2. Bundler (`gem install bundler`)
+3. Jekyll (`gem install jekyll`)
+4. Once Bundler and Jekyll gems are installed, run `bundle install`
+
+#### Run Locally
+To construct:
+1. `git clone https://github.com/cyberark/secretless-broker`
+2. `cd docs`
+3. Run the following command:
+`bundle exec jekyll serve`
+4. Preview Jekyll site locally in web browser by either running `open localhost:4000` or manually navigating to http://localhost:4000
+
+#### Run in Docker
+With `docker` and `docker-compose`:
+
+1. Run `docker-compose up -d` in the `docs` directory.
+2. Preview Jekyll site locally in web browser by either running `open localhost:4000` or manually navigating to http://localhost:4000
+
```

Comment:
Let's note that with both method, changes will be picked up without a server restart.